<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="6" passed="5">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="106" started-at="2019-06-07T10:55:47Z" finished-at="2019-06-07T10:55:47Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="106" started-at="2019-06-07T10:55:47Z" finished-at="2019-06-07T10:55:47Z">
      <class name="Assertion.HardAssertion">
        <test-method status="FAIL" signature="assertEquals()[pri:0, instance:Assertion.HardAssertion@28feb3fa]" name="assertEquals" duration-ms="19" started-at="2019-06-07T10:55:47Z" finished-at="2019-06-07T10:55:47Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [This assertion will fail] but found [assertion]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [This assertion will fail] but found [assertion]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at Assertion.HardAssertion.assertEquals(HardAssertion.java:11)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- assertEquals -->
        <test-method status="PASS" signature="assertFalse()[pri:0, instance:Assertion.HardAssertion@28feb3fa]" name="assertFalse" duration-ms="2" started-at="2019-06-07T10:55:47Z" finished-at="2019-06-07T10:55:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- assertFalse -->
        <test-method status="PASS" signature="assertNotEquals()[pri:0, instance:Assertion.HardAssertion@28feb3fa]" name="assertNotEquals" duration-ms="2" started-at="2019-06-07T10:55:47Z" finished-at="2019-06-07T10:55:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- assertNotEquals -->
        <test-method status="PASS" signature="assertNotNull()[pri:0, instance:Assertion.HardAssertion@28feb3fa]" name="assertNotNull" duration-ms="3" started-at="2019-06-07T10:55:47Z" finished-at="2019-06-07T10:55:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- assertNotNull -->
        <test-method status="PASS" signature="assertNull()[pri:0, instance:Assertion.HardAssertion@28feb3fa]" name="assertNull" duration-ms="3" started-at="2019-06-07T10:55:47Z" finished-at="2019-06-07T10:55:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- assertNull -->
        <test-method status="PASS" signature="assertTrue()[pri:0, instance:Assertion.HardAssertion@28feb3fa]" name="assertTrue" duration-ms="3" started-at="2019-06-07T10:55:47Z" finished-at="2019-06-07T10:55:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- assertTrue -->
      </class> <!-- Assertion.HardAssertion -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
